frontend/src/components/labwork/step/LabworkStep.tsx

```diff
@@ -38,6 +39,31 @@ const LabworkStep = ({ protocol, step, stepSamples }: LabworkStepPageProps) => {
 	const SAMPLES_TAB_KEY = 'samples'
 	const SELECTION_TAB_KEY = 'selection'
 	const [selectedTab, setSelectedTab] = useState<string>(SAMPLES_TAB_KEY)
+	const samplesByID = useAppSelector(selectSamplesByID)
+	const librariesByID = useAppSelector(selectLibrariesByID)
+	const [samples, setSamples] = useState<SampleAndLibrary[]>([])
+	const [selectedSamples, setSelectedSamples] = useState<SampleAndLibrary[]>([])
+
+	const getSampleList = (sampleIDs) => {
```

Comment:
You should put this function inside the useEffect that is calling it, since it isn't called anywhere else. I am getting a linter error because `getSampleList` is not listed in the dependencies array of the useEffect. Moving the function definition inside the useEffect will solve that.